src/VirtualMonoRepo/InitializeVMR.proj

```diff
@@ -0,0 +1,121 @@
+<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. -->
+<Project>
+  <PropertyGroup>
+    <ArtifactsDir>$(RepoRoot)artifacts/</ArtifactsDir>
+    <VirtualMonoRepoTasksAssembly>$(ArtifactsDir)bin/VirtualMonoRepo.Tasks/$(Configuration)/net7.0/VirtualMonoRepo.Tasks.dll</VirtualMonoRepoTasksAssembly>
+
+    <VmrDir Condition=" '$(VmrDir)' != '' ">$([MSBuild]::EnsureTrailingSlash('$(VmrDir)'))</VmrDir>
+    <TmpDir Condition=" '$(TmpDir)' == '' ">$(RepoRoot)tmp/</TmpDir>
+    <VmrDefaultBranch Condition=" '$(VmrDefaultBranch)' == '' ">main</VmrDefaultBranch>
```

Comment:
I don't love defaulting here. It leaves open too many possibilities that the default will get used and accidentally expose sensitive source. If we have a default, I'd say we should use the AzDO source branch (`Build.SourceBranch` IIRC) and error if there is no value specified.

Reply:
This is just the name of the branch the new VMR gets created with (passed to `git init`), not the default for `source-mappings`